packages/riipen-ui/src/components/ClickAway.test.jsx

```diff
@@ -0,0 +1,114 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import ClickAway from "./ClickAway";
+
+describe("<ClickAway>", () => {
+  it("renders without errors", () => {
+    let error;
+    const onClickAway = jest.fn();
+
+    try {
+      mount(<ClickAway onClickAway={onClickAway} />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const onClickAway = jest.fn();
+
+    const wrapper = mount(<ClickAway onClickAway={onClickAway} />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("renders given non null children nodes", () => {
+      const child = <span>cheesecakes</span>;
+      const onClickAway = jest.fn();
+
+      const wrapper = mount(
+        <ClickAway onClickAway={onClickAway}>
+          {null}
+          {child}
+        </ClickAway>
+      );
+
+      const childrenNodes = wrapper
+        .find("ClickAway")
+        .childAt(0)
+        .children();
+      expect(childrenNodes.at(0).containsMatchingElement(child)).toEqual(true);
+    });
+  });
+
+  describe("onClickAway prop", () => {
+    it("sets onClick prop of children node", () => {
+      const child = <span>cheesecakes</span>;
+      const onClickAway = jest.fn();
+
+      const wrapper = mount(
+        <ClickAway onClickAway={onClickAway}>{child}</ClickAway>
+      );
+
+      expect(
+        wrapper
+          .find("ClickAway")
+          .childAt(0)
+          .children()
+          .at(0)
+          .props().onClick
+      ).toBeDefined();
+    });
+
+    it("sets onClick prop of div element", () => {
```

Comment:
This test isn't really testing anything to do with `onClickAway` prop. I think we can actually remove this test entirely because it is actually covered by the test on line 81.